### PR TITLE
Feat/niloufar mont

### PR DESCRIPTION
**Pull request by lilmont**
I tried to find a way to not copy the batch file in the new folder created, but I could not find a way. I'd really like to know if there's any other option.
Also if mha-builder is executed outside my project root, it'll return error code 1.

### CODE REVIEW

File: mha-builder/mha-builder.csproj [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+	  <PackAsTool>true</PackAsTool>
+	  <ToolCommandName>mha-builder</ToolCommandName>
+	  <PackageOutputPath>./pkg</PackageOutputPath>
```
Comment (thisisnabi) [inline mha-builder/mha-builder.csproj:8]: Here, you can use `<Version>1.0.0</Version>` tag for managing version of your assembly.

File: mha-builder/Program.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
+﻿using System.Diagnostics;
+
+if (args.Length > 0 && args[0] == "--name")
+{
+    if (args.Length > 1 && args[1]!=null)
```
Comment (thisisnabi) [inline mha-builder/Program.cs:5]: You can use `CommandLineParser` For parsing your arguments in console apps.

File: mha-builder/mha-builder.csproj (hunk 1 above)
Comment (thisisnabi) [inline mha-builder/mha-builder.csproj:8]: Here, you can use `<Version>1.0.0</Version>` tag for managing version of your assembly.

**Review by thisisnabi — APPROVED** [review]
Thanks Nilo, It was be great implementation.

**Comment by thisisnabi** [comment]
@lilmont  Please move the maha-builder app into the your-self folder.

File: mha-builder/mha-builder.csproj (hunk 1 above)
Comment (lilmont) [reply mha-builder/mha-builder.csproj:8]: I added it. How can I handle it? Should I do it manually?

File: mha-builder/Program.cs (hunk 2 above)
Comment (lilmont) [reply mha-builder/Program.cs:5]: Done :)

**Comment by thisisnabi** [comment]
image
Please do this @lilmont

**Comment by thisisnabi** [comment]
Please do this @lilmont 

image

**Pull request merged**